### ISSUE TITLE
Add loss function as a hyperparameter

### ISSUE THREAD

**Issue by vlandau** [csp-inc/fluvius#22]
Consider additional loss functions for continuous-response data, per the fluvius work plan. This should be added as a hyperparameter to both MLP and CNN functions and tracked in the grid search.
